src/Query/Explain/Explanation.ts

```diff
@@ -61,14 +61,24 @@ export class Explanation {
             return currentIndentation + this.description;
         }
 
-        // We have children, so concatenate them together
-        let result = currentIndentation + `${this.symbol}`;
-        if (this.children.length > 1) {
-            // The descriptions like 'All of', 'None of' are one really meaningful
-            // if there is more than one filter. Otherwise, they are just confusing.
-            result += ` (${this.description})`;
+        let result = currentIndentation;
+
+        if (this.symbol === '') {
+            // If the symbol was not set, add the description and just indent
+            result += this.description;
+        } else {
+            // Otherwise we need detailed explanation with logic
+            result += this.symbol;
+
+            // We have children, so concatenate them together
+            if (this.children.length > 1) {
+                // The descriptions like 'All of', 'None of' are one really meaningful
+                // if there is more than one filter. Otherwise, they are just confusing.
+                result += ` (${this.description})`;
+            }
+            result += ':';
```

Comment:
Based on the behaviour, which I've tested, I would totally merge this PR.

I just wonder whether this new behaviour in `Explanation` would benefit from being tested in `Explanation.test.ts`?

I'm assuming it wasn't already covered by existing tests there, as no tests failed.

Reply:
oh, I actually went the other way around and checked the `Explanation.asString()` calls =)

In any case you are totally right, I just added a test case for this scenario in `Explain.test`. Thanks for noticing this.

If the test is ok pls resolve =)

Reply:
Excellent - line coverage on `Explanation.asString()` is now 100%.

Very many thanks. Resolving comment.